KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -470,6 +470,8 @@ public enum UITexts {
         static let info = "Text.Loyalty.Info".localized
         static let noAllowedToBurnPoints = "Text.Loyalty.BurnNotAllowed".localized
         static let balanceTitle = "Text.Loyalty.BalanceTitle".localized
+        static let loyaltyPointsRemovedInfo = "Text.Loyalty.PointsRemovedInfo".localized
```

Comment:
please create new one in "Android" format and add your keys to Android ticket: https://karhoo.atlassian.net/browse/MOB-4443
We need to keep new keys aligned with new format, while old one are ready to merge in proper Mihai's PR.

Reply:
Correct format is like: `kh_uisdk_address_picker_enter_destination`
- with "kh_uisdk_"prefix
- lowercase
- with underscore